src/styles.css

```
@@ -2158,15 +2158,15 @@ body {
   align-items: center;
   justify-content: space-between;
   padding: 6px 10px;
-  background: var(--vscode-code-bg);
-  border: 1px solid var(--vscode-border);
+  background: var(--liminis-editor-code-bg, var(--vscode-code-bg));
+  border: 1px solid var(--liminis-editor-border, var(--vscode-border));
   border-radius: 4px;
 }
 
 .mdc-glob-pattern {
   font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)